yfinance/base.py

```diff
@@ -315,8 +315,14 @@ def cleanup(data):
             if isinstance(data.get(item), dict):
                 self._info.update(data[item])
 
-        self._info['regularMarketPrice'] = self._info['regularMarketOpen']
+
+        try:
+            self._info['regularMarketPrice'] = self._info['regularMarketOpen']
+        except Exception:
+            pass                
```

Comment:
Maybe assign a None to the self._info['regularMarketPrice'] when exception is caught instead of pass?

Reply:
The thing is that the rest of the code does exactly the same:

      except Exception:
            pass       

I want to align with the rest of the code

Exception is good since outside (the caller) can simply test it:

        if 'regularMarketPrice' in info:
            value = info['regularMarketPrice']
        else:
            value = REGULAR_MARKET_UNKNOWN  # Or None, or whatever the caller needs for that matter

Reply:
Example from the "rest of the code" I mentioned (notice the similar `except Exception: pass` usage):

        try:
            cal = _pd.DataFrame(
                data['calendarEvents']['earnings'])
            cal['earningsDate'] = _pd.to_datetime(
                cal['earningsDate'], unit='s')
            self._calendar = cal.T
            self._calendar.index = utils.camel2title(self._calendar.index)
            self._calendar.columns = ['Value']
        except Exception:
            pass

        # analyst recommendations
        try:
            rec = _pd.DataFrame(
                data['upgradeDowngradeHistory']['history'])
            rec['earningsDate'] = _pd.to_datetime(
                rec['epochGradeDate'], unit='s')
            rec.set_index('earningsDate', inplace=True)
            rec.index.name = 'Date'
            rec.columns = utils.camel2title(rec.columns)
            self._recommendations = rec[[
                'Firm', 'To Grade', 'From Grade', 'Action']].sort_index()
        except Exception:
            pass